cms/static/js/index.js

```diff
@@ -169,7 +174,22 @@ define(['domReady', 'jquery', 'underscore', 'js/utils/cancel_on_escape', 'js/vie
             };
         };
 
+        var setupSecondaryOrgAutocomplete = function(e) {
+            $('.new-course-secondary-org').empty();
+            $.each(e.target.options, function( i, option ){
```

Comment:
Please run some formatter or linter to have the formatting. Like the spaces here.
```suggestion
            $.each(e.target.options, function( i, option ) {
```